packages/install-ses/install-ses.js

```diff
@@ -1,11 +1,12 @@
 /* global lockdown */
+
 // 'lockdown' appears on the global as a side-effect of importing 'ses'
 import 'ses';
 
-// we need to enable Math.random as a workaround for 'brace-expansion' module
-// (dep chain: cosmic-swingset/ag-solo->temp->glob->minimatch->brace-expansion)
-// we need Date.now to build a timer device for cosmic-swingset
-lockdown({ mathTaming: 'unsafe', dateTaming: 'unsafe', errorTaming: 'unsafe' });
+// Install our HandledPromise global.
+import '@agoric/eventual-send/shim';
+
+lockdown({ errorTaming: 'unsafe' });
```

Comment:
@michaelfig I'm surprised this works (and it seems unrelated to anything involving `HandledPromise`), what changed to allow either untaming to be removed?

Reply:
The modern SES start compartment now contains the original `Math.random` and `Date.now`.

Reply:
oh, awesome, I missed that. That's the perfect solution.